### PR TITLE
Feature/actix web example

### PR DESCRIPTION
**Pull request by biryukovmaxim**

### CODE REVIEW

File: actix-web/hello-world/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+use shuttle_service::ShuttleActixWeb;
+
+async fn hello_world() -> &'static str {
+    "Hello World!"
+}
+
+#[shuttle_service::main]
+async fn actix_web(
+) -> ShuttleActixWeb<impl FnOnce(&mut ServiceConfig) + Sync + Send + Copy + Clone + 'static> {
+    let h = hello_world;
```
Comment (chesedo) [inline actix-web/hello-world/src/lib.rs:11]: Will this be needed for all routes? Ie assigning the pointer to a variable first?

**Review by chesedo — APPROVED** [review]
This seems good to me. I'm just wondering if the example can be simplified?

File: actix-web/hello-world/src/lib.rs (hunk 1 above)
Comment (biryukovmaxim) [reply actix-web/hello-world/src/lib.rs]: No, it's not required. Actually it was part of my experiments with closures. Fix pushed

File: actix-web/hello-world/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "hello-world"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+actix-service = "2.0.2"
+actix-web = "4.2.1"
+shuttle-service = { version = '0.7.2', git = "ssh://git@github.com/biryukovmaxim/shuttle.git", branch = "feature/support-actix-web", features = ["web-actix-web"] }
```
Comment (oddgrd) [inline actix-web/hello-world/Cargo.toml:9]: Looks good to me too! We should replace this so it's ready for release. If you want to test the example before release, you can use a `.cargo/config.toml` file in the root of the shuttle repo so it uses the current version of `shuttle-service`. There is an example of this [here](https://github.com/shuttle-hq/shuttle/blob/main/CONTRIBUTING.md#running-locally)
```suggestion
shuttle-service = { version = '0.7.2', features = ["web-actix-web"] }
```

**Review by oddgrd — APPROVED** [review]
LGTM too, just one small suggestion. Thanks!

**Pull request merged**